frontend/src/App.tsx

```diff
@@ -1,11 +1,37 @@
 import * as React from "react";
+import { Layout, Row, Col } from "antd";
+import { ApartmentOutlined } from "@ant-design/icons";
+const { Header, Content, Footer } = Layout;
+import { Chain, InitialBlock } from "./components";
 
-import { ANTVersion } from "./components";
+// const mockBlocks: string[] = [];
+
+// To see with state use the following:
+const mockBlocks: string[] = ["#FF008C", "#D309E1", "#9C1AFF", "#7700FF"];
 
 const App: React.FC = () => (
-  <div>
-    <ANTVersion />
-  </div>
+  <Layout className="layout">
+    <Header>
+      <Row gutter={10}>
+        <Col>
+          <ApartmentOutlined style={{ marginTop: "10px", fontSize: "40px" }} />
```

Comment:
Is this the suggested way of styling Ant components or can we write actual CSS or styled-components or w/e instead? Is that possible?

Reply:
Yeah we can write actual CSS or use styled-components.